<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendNotification</class>
 <widget class="QDialog" name="sendNotification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox{
font: 60 12pt &quot;Segoe UI Light&quot;;
}
QPushButton{
font: 25 12pt &quot;Segoe UI Light&quot;;
color: white;
background-color: rgba(0, 0,0,0.5);
border-radius: 10px;
}
QPushButton::hover{
font: 25 12pt &quot;Segoe UI Light&quot;;
color: white;
background-color: rgba(30, 30,30,0.5);
border-radius: 10px;
}
QPushButton::pressed{
font: 25 12pt &quot;Segoe UI Light&quot;;
color: white;
background-color: rgba(80, 80,80,0.5);
border-radius: 10px;
}
#sendNotification{
border-image: url(:/appscreen/Resources/appscreen/Admin_Background.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>831</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background-color: rgba(255, 255, 255,0.5);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI Symbol</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
background:rgba(255, 255, 255,0);
font: 11pt &quot;Segoe UI Symbol&quot;;
</string>
         </property>
         <property name="text">
          <string>   To whom would you want to send message?(separate with space(s))</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>398</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="sendLE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>831</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{
background-color: rgba(255,255, 255,0.5);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: black;
background:rgba(255, 255, 255,0);
font: 11pt &quot;Segoe UI Symbol&quot;;
</string>
       </property>
       <property name="text">
        <string>    ...or to which group of people would you want to send?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>60</number>
       </property>
       <item>
        <widget class="QCheckBox" name="readerCB">
         <property name="styleSheet">
          <string notr="true">color: black;
background:rgba(255, 255, 255,0)</string>
         </property>
         <property name="text">
          <string>Reader</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="librarianCB">
         <property name="styleSheet">
          <string notr="true">color: black;
background:rgba(255, 255, 255,0)</string>
         </property>
         <property name="text">
          <string>Librarian</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dataProcessorCB">
         <property name="styleSheet">
          <string notr="true">color: black;
background:rgba(255, 255, 255,0)</string>
         </property>
         <property name="text">
          <string>Data Processor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="administratorCB">
         <property name="styleSheet">
          <string notr="true">color: black;
background:rgba(255, 255, 255,0)</string>
         </property>
         <property name="text">
          <string>Administrator</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>831</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3{
background-color: rgba(255, 255, 255,0.7);
border-radius:10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="messageTE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
